### DISCUSSION TITLE
help me!    minio  invalid Login.

### DISCUSSION THREAD

**Discussion by waymen** [minio/minio#19678] [Q&A]
### os version
```
centos 7.9
```

### minio version:

```
# minio --version
minio version RELEASE.2024-05-01T01-11-10Z (commit-id=7926401cbd5cceaacd9509f2e50e1f7d636c2eb8)
Runtime: go1.21.9 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2024 MinIO, Inc.
```

### minio env
```
MINIO_ROOT_USER="minioadmin"
MINIO_ROOT_PASSWORD="minioadmin@123"
MINIO_VOLUMES="http://minio-{1...4}:9000/data/minio/drive-{1...4}"
MINIO_VOLUMES="/data/minio"
MINIO_SERVER_URL="http://minio-cluster-dev.fsyhlz.com"
MINIO_OPTS="--certs-dir /data/minio/.minio/certs --address :9000 --console-address :9001"
```

### login error
when i use minioadmin/minioadmin@123 login to minio

![image](https://github.com/minio/minio/assets/16431201/85f516dd-9d38-429c-bb6e-265b12c5a2fc)

**Comment by pingchunzhang** [comment]
do u have solved this problem？

**Reply by llchry** [reply]
I have not had this problem in my use. I think you need to provide more information. and you need to check your run logs.

**Comment by chenjinkai** [comment]
i have the same problem. how to?? sading....

**Reply by bytescat** [reply]
i have the same problem. "minio: Error: Prefix access is denied: .minio.sys/config/iam/sts/11ORRTIJJGTG94SR5H6K/identity.json (cmd.PrefixAccessDenied)"
